x-pack/solutions/observability/plugins/infra/public/services/telemetry/types.ts

```diff
@@ -95,6 +100,12 @@ export interface AnomalyDetectionFilterFieldChangeParams {
   filter_field?: string;
 }
 
+export interface SchemaSelectorParams {
+  interaction: string;
```

Comment:
Could we enforce a more strict types?  `interaction: 'open dropdown' | 'select schema'`